content/guides/weird_characters.adoc

```diff
@@ -0,0 +1,846 @@
+= The Weird and Wonderful Characters of Clojure
+James Hughes
+2017-05-27
+:type: guides
+:toc: macro
+
+ifdef::env-github,env-browser[:outfilesuffix: .adoc]
+
+[]
+====
+This is a reference collection of characters used in Clojure that are difficult to "google".
+Descriptions are sourced from various blogs, https://stackoverflow.com[StackOverflow],
+http://en.wikibooks.org/wiki/Learning_Clojure[Learning Clojure], and the
+http://clojure.org/documentation[official Clojure docs] -- sources attributed
+where necessary. Sections are not in any particular order, but related items
+are grouped for ease. This guide is a copy of http://twitter.com/kouphax[James Hughes]
+brilliant https://yobriefca.se/blog/2014/05/19/the-weird-and-wonderful-characters-of-clojure/[blog post] of the same name.
+====
+[[dispatch]]
+== `#` - Dispatch macro
```

Comment:
I think it confusing to call this the "dispatch macro", especially without distinguishing between reader macros and compiler macros. What if we called this the "dispatch character"?